### PR TITLE
Docs: add missing alt text - transformations 10.3 (2/3)

### PR DESCRIPTION
**Pull request by imatwawana**
Adds missing alt text in Transform Data in 10.3.

**TO DO**

- [x] Add alt text to files in content.ts

### CODE REVIEW

**Comment by baldm0mma** [comment]
@imatwawana `content.js` alt text added. :)

**Review by baldm0mma — APPROVED** [review]
LGTM.

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ Use this transformation to add a new field calculated from two other fields. Eac
 
 In the example below, we added two fields together and named them Sum.
 
-{{< figure src="/static/img/docs/transformations/add-field-from-calc-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/add-field-from-calc-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Add field from calculation docs image" >}}
```
[the remark this answers is not in the document]
Comment (baldm0mma) [reply docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:160]: Yeah, if you want to make the changes/updates to the alt text in the markdown that you think fits, I can update/rebuild accordingly.

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 2]
```diff
@@ -384,7 +384,7 @@ Grafana displays the query identification letters in dark gray text. Click a que
 
 In the example below, the panel has three queries (A, B, C). We removed the B query from the visualization.
 
-{{< figure src="/static/img/docs/transformations/filter-by-query-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/filter-by-query-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Filter data by query refId docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:387]:
```suggestion
{{< figure src="/static/img/docs/transformations/filter-by-query-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="A stat visualization with results from two queries, A and C" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 3]
```diff
@@ -773,11 +773,11 @@ The result after applying the outer join transformation looks like the following
 
 In the following example, a template query displays time series data from multiple servers in a table visualization. The results of only one query can be viewed at a time.
 
-{{< figure src="/static/img/docs/transformations/join-fields-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/join-fields-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Join by field 1 docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:776]:
```suggestion
{{< figure src="/static/img/docs/transformations/join-fields-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="A table visualization showing results for one server" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 4]
```diff
@@ -773,11 +773,11 @@ The result after applying the outer join transformation looks like the following
...
 
 I applied a transformation to join the query results using the time field. Now I can run calculations, combine, and organize the results in this new table.
 
-{{< figure src="/static/img/docs/transformations/join-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/join-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Join by field 2 docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:780]:
```suggestion
{{< figure src="/static/img/docs/transformations/join-fields-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="A table visualization showing results for multiple servers" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 5]
```diff
@@ -1117,11 +1117,11 @@ You can specify a regular expression, which is only applied to matches, along wi
 
 In the following example, we are stripping the prefix from event types. In the before image, you can see everything is prefixed with 'system.'
 
-{{< figure src="/static/img/docs/transformations/rename-by-regex-before-7-3.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/rename-by-regex-before-7-3.png" class="docs-image--no-shadow" max-width= "1100px" alt="Rename by regex 1 docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:1120]:
```suggestion
{{< figure src="/static/img/docs/transformations/rename-by-regex-before-7-3.png" class="docs-image--no-shadow" max-width= "1100px" alt="A bar chart with long series names" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 6]
```diff
@@ -1117,11 +1117,11 @@ You can specify a regular expression, which is only applied to matches, along wi
...
 
 With the transformation applied, you can see we are left with just the remainder of the string.
 
-{{< figure src="/static/img/docs/transformations/rename-by-regex-after-7-3.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/rename-by-regex-after-7-3.png" class="docs-image--no-shadow" max-width= "1100px" alt="Rename by regex 2 docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:1124]:
```suggestion
{{< figure src="/static/img/docs/transformations/rename-by-regex-after-7-3.png" class="docs-image--no-shadow" max-width= "1100px" alt="A bar chart with shortened series names" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 7]
```diff
@@ -489,15 +489,15 @@ Click and uncheck the field names to remove them from the result. Fields that ar
 
 Enable 'From variable' to let you select a dashboard variable that's used to include fields. By setting up a [dashboard variable][] with multiple choices, the same fields can be displayed across multiple visualizations.
 
-{{< figure src="/static/img/docs/transformations/filter-name-table-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/filter-name-table-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Filter fields by name 1 docs image" >}}
 
 Here's the table after we applied the transformation to remove the Min field.
 
-{{< figure src="/static/img/docs/transformations/filter-name-table-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/filter-name-table-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Filter fields by name 2 docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:496]:
```suggestion
{{< figure src="/static/img/docs/transformations/filter-name-table-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="A table visualization with value and Max columns" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 8]
```diff
@@ -489,15 +489,15 @@ Click and uncheck the field names to remove them from the result. Fields that ar
 
 Enable 'From variable' to let you select a dashboard variable that's used to include fields. By setting up a [dashboard variable][] with multiple choices, the same fields can be displayed across multiple visualizations.
 
-{{< figure src="/static/img/docs/transformations/filter-name-table-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/filter-name-table-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Filter fields by name 1 docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:492]:
```suggestion
{{< figure src="/static/img/docs/transformations/filter-name-table-before-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="A table visualization with time, value, Min, and Max fields" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 9]
```diff
@@ -489,15 +489,15 @@ Click and uncheck the field names to remove them from the result. Fields that ar
...
 
 Here's the table after we applied the transformation to remove the Min field.
 
-{{< figure src="/static/img/docs/transformations/filter-name-table-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/filter-name-table-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Filter fields by name 2 docs image" >}}
 
 Here is the same query using a Stat visualization.
 
-{{< figure src="/static/img/docs/transformations/filter-name-stat-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+{{< figure src="/static/img/docs/transformations/filter-name-stat-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="Filter fields by name 3 docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:500]:
```suggestion
{{< figure src="/static/img/docs/transformations/filter-name-stat-after-7-0.png" class="docs-image--no-shadow" max-width= "1100px" alt="A stat visualization with value and Max fields" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 10]
```diff
@@ -1273,9 +1273,9 @@ Use this transformation to create a new data frame containing values predicted b
 There are two different models:
 
 - **Linear regression** - Fits a linear function to the data.
-  {{< figure src="/static/img/docs/transformations/linear-regression.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+  {{< figure src="/static/img/docs/transformations/linear-regression.png" class="docs-image--no-shadow" max-width= "1100px" alt="Linear regression docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:1276]:
```suggestion
  {{< figure src="/static/img/docs/transformations/linear-regression.png" class="docs-image--no-shadow" max-width= "1100px" alt="A time series visualization with a line representing the linear function" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 11]
```diff
@@ -1273,9 +1273,9 @@ Use this transformation to create a new data frame containing values predicted b
...
 - **Polynomial regression** - Fits a polynomial function to the data.
-  {{< figure src="/static/img/docs/transformations/polynomial-regression.png" class="docs-image--no-shadow" max-width= "1100px" >}}
+  {{< figure src="/static/img/docs/transformations/polynomial-regression.png" class="docs-image--no-shadow" max-width= "1100px" alt="Polynomial regression docs image" >}}
```
Comment (imatwawana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:1278]:
```suggestion
  {{< figure src="/static/img/docs/transformations/polynomial-regression.png" class="docs-image--no-shadow" max-width= "1100px" alt="A time series visualization with a curved line representing the polynomial function" >}}
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:160]: @baldm0mma - I've added suggestions for all the images. Can you take a look and make sure they're all accurate? If you're okay with them, just go ahead and commit them and then update/rebuild.

**Pull request merged**